packages/rest/src/rest.application.ts

```diff
@@ -133,6 +133,11 @@ export class RestApplication extends Application implements HttpServerLike {
     this.restServer.basePath(path);
   }
 
+  controller<T>(controllerCtor: ControllerClass<T>, name?: string): Binding {
+    this.restServer.invalidateRoutingCache();
```

Comment:
I would rather to use a `ContextView` to invalidate the routing cache.

Reply:
+1, I would like to see a ContextView-based solution too. In this spike, I used the fastest path to get done what I need.

@raymondfeng Do you have bandwidth to implement ContextView-based version yourself? Here is the GitHub issue tracking this user story: https://github.com/strongloop/loopback-next/issues/433